packages/client/auth-core/src/utils/constants.ts

```diff
@@ -0,0 +1,11 @@
+// testing purposes
+export const CROSSMINT_STG_URL = "http://localhost:3000";
+// export const CROSSMINT_STG_URL = "https://staging.crossmint.com";
```

Comment:
Should we leave this instead?
Maybe we could find a way to inject the testing values without committing them

Reply:
We should remove it and add it when testing/developing, for sure!